src/components/organisms/UiModal/UiModal.vue

```diff
@@ -541,17 +526,16 @@ onBeforeUnmount(() => {
     flex-direction: column;
 
     @include mixins.from-tablet {
-      flex-direction: row;
       justify-content: flex-end;
+      display: grid;
+      grid-template-areas: "cancel confirm";
```

Comment:
Couldn't we achieve that by just `display: flex` and `flex: 1 1 0` on every child? The grid seems to be overkill for this use case as it's mainly used for 2D layouts, whereas flex is mainly used for 1D layouts.

Reply:
I left `grid` because your solution needs more code than only these two lines (setting the width, align at the end, in one line)